client/src/components/v2/charts/LocationTimeChart.module.css

```diff
@@ -0,0 +1,9 @@
+.map {
+  height: 100%;
+}
+
+@media (max-width: 991.98px) {
```

Comment:
Where did 991.98px come from? (Just curious)

Reply:
This is the [bootstrap breakpoint for medium devices](https://getbootstrap.com/docs/4.0/layout/overview/). I'll leave a comment to clarify

Reply:
Ohhhh. Yes please leave a comment.